fr/basic-concepts/environments-to-organize-workloads-and-users.md

```diff
@@ -3,14 +3,48 @@ title: "Les environnements pour compartimenter charges de travail et utilisateur
 
 Bien que CloudMC mesure l'utilisation des services au niveau des organisations pour des fins de facturation, les ressources consommées par chaque environnement sont également mesurées indépendamment, ce qui permet au entreprises qui le désirent d'effectuer de la facturation interne par environnement.
 
-Rôles d'environnement
-À chaque environnement dont un utilisateur est membre, corresponds aussi un **rôle d’environnement** qui contrôle ce qu’il a le droit de voir et les opérations qu’il est autorisé à executer à l’intérieur de cet environnement. Contrairement aux [rôles système](system-roles.md), les rôles d'environnement sont fixes et ne peuvent pas (encore) être configurés de façon granulaire.
+Naviguez au service souhaité dans la barre latérale pour accéder à la page *Environnements*, où tous les environnements du service sélectionné sont répertoriés. De plus, lorsque vous travaillez à l'intérieur d'un environnement, le service et l'environnement actuels sont affichés en haut à gauche de la page. Cliquer sur ce bouton affichera une option pour revenir à la page *Environnements*, ou pour basculer vers un autre environnement.
+
+### Rôles d'environnement
+
+L'appartenance à l'environnement est associée à un **rôle d'environnement**, qui contrôle ce qu'un utilisateur peut faire avec les ressources contenues dans cet environnement. Contrairement aux rôles système, les rôles d'environnement sont fixes et ne peuvent pas être personnalisés de manière granulaire.
+
+### Créer un environnement
+
+Tout utilisateur ayant le rôle **Utilisateur**, ou toute autre rôle disposant de la permission **Environnement : Créer** peut ajouter de nouveaux environnements et décider des personnes qui y auront accès.
+
+1. Cliquez sur *Ajouter environnement*. La page *Ajouter environnement* apparaît.
+1. Saisissez un nom et une description facultative pour l'environnement.
+1. Si vous souhaitez autoriser des utilisateurs d'autres organisations à être ajoutés en tant que membres, sélectionnez **Autoriser les membres externes**.
+1. Cliquez sur *Suivant*. L'environnement sera approvisionné.
+1. La page *Gérer les membres* apparaît. Ici, vous pouvez :
+    - Sélectionnez un ou plusieurs utilisateurs individuels dans la liste
+    - Rechercher d'utilisateurs spécifiques
+    - Ajouter **tous les utilisateurs** dans l'organisation (**géstion automatique**)
+    - N'ajoutez aucun utilisateur à votre environnement (cliquez sur *Sauter* pour passer à la page suivante)
+1. Pour tous les utilisateurs que vous avez ajoutés, sélectionnez un rôle d'environnement. Si vous avez activé la géstion automatique, sélectionnez un rôle d'environnement par défaut.
+1. Cliquez sur *Appliquer* pour valider les membres sélectionnés. Les utilisateurs seront ajoutés.
+1. La page *Initialiser l'environnement* apparaît. Ici, vous pouvez :
```

Comment:
same as the previous comment about apparaît.
```suggestion
1. La page *Initialiser l'environnement* apparaîtra. Ici, vous pouvez :
```

Reply:
I will make the same changes as above.